module/PowerShellEditorServices/Commands/Public/CmdletInterface.ps1

```diff
@@ -78,10 +78,38 @@ function Unregister-EditorCommand {
 }
 
 function Open-EditorFile {
-    param([Parameter(Mandatory=$true)]$FilePaths)
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        $FilePaths,
+
+        [Parameter(ValueFromPipeline=$true)]
+        $NewValue,
```

Comment:
[Set-Content](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-content?view=powershell-5.1) uses `Value` as the name for this parameter, maybe good to be consistent since the behavior is similar?  Not really a big deal, though.